Rubjerg.Graphviz/ForeignFunctionInterface.cs

```diff
@@ -143,7 +143,28 @@ public static void Agset(IntPtr obj, string name, string value)
             {
                 agset(obj, name, value);
             }
+        }   
+        
+        public static void AgreLabelNode(IntPtr obj, string value)

```

Comment:
I think a name like `AddLabel` would be clearer. The Ag prefix is only taken over from functions that are more or less directly exposed (modulo the mutex). And this function would work for any object, not just nodes, correct?

Reply:
Yes, you are right. Agreed and updated